### DISCUSSION TITLE
The Need for Up-to-Date Knowledge in GitHub Copilot for Flutter Developers

### DISCUSSION THREAD

**Discussion by fisforfaheem** [community/community#126652] [Copilot Conversations]
Main Post:

Current model urgently needs UPDATED KNOWLEDGE. Currently, it provides data from 2021, which is considerably outdated and inadequate for Flutter and Dart development.

This outdated information poses significant challenges for developers who rely on the latest features and best practices introduced in Flutter 3.

It is crucial that the model incorporates the latest advancements in Flutter 3. ChatGPT 4 already includes this updated knowledge, but your current model does not.

Can you please address why this gap exists and ensure that the model is updated with the most recent information to better support developers?

**Questions for Discussion:**

1. **Has anyone else experienced difficulties due to outdated information in GitHub Copilot, particularly with Flutter and Dart?**
2. **What specific features or updates from Flutter 3 do you find missing in the current model?**
3. **Are there any workarounds you have found to mitigate the lack of up-to-date information?**
4. **What steps can we, as a community, take to ensure our tools remain current and effective for development?**
5. **How often should these models be updated to keep pace with rapid advancements in technology like Flutter?**

Looking forward to hearing your thoughts and experiences!

**Comment by ivan23z** [comment]
Cuento con el mismo problema, falta de información actualizada.

**Comment by fisforfaheem** [comment]
+1
